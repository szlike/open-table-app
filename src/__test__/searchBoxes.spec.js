import React from "react"
import { create } from "react-test-renderer"


function dummyFunction (){
  return 'do nothing'
}

const cityName = 'Toronto_test'

describe("SearchBoxes component", () => {
  test("Matches the snapshot", () => {
    const SearchBoxes = create(<searchBoxes />)
    expect(SearchBoxes.toJSON()).toMatchSnapshot()
  })

  test("it pass correct user input when clicked ", () => {
    const SearchBoxes = create(<searchBoxes searchByCity={dummyFunction}
      cityName={cityName}  />)
    const instance = SearchBoxes.root
    const button = instance.findByType("input")
    button.props.onClick()
    expect(instance.state.text).toBe("Toronto_test")
  })
})

