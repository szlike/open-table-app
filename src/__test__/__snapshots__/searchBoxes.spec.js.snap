// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBoxes component Matches the snapshot 1`] = `<searchBoxes />`;
